lib/gui/models/store.js

```diff
@@ -94,15 +95,15 @@ const storeReducer = (state, action) => {
 
         const drive = _.first(action.data);
 
-        if (_.every([
-
-          // TODO: Reuse from SelectionStateModel.isDriveValid()
-          state.getIn([ 'selection', 'image', 'size' ], 0) <= drive.size,
+        // Even if there's no image selectected, we need to call `isDriveValid`
+        // to check if the drive is locked, and `{}` works fine with it
+        const image = state.getIn([ 'selection', 'image' ], {
```

Comment:
`state.getIn([ 'selection', 'image' ], {});` will return `{}` when the image is not in the store, and calling `({}).toJS()` will throw a `TypeError`